src/node.cc

```diff
@@ -905,12 +905,21 @@ Local<Value> UVException(Isolate* isolate,
 
 
 // Look up environment variable unless running as setuid root.
-inline const char* secure_getenv(const char* key) {
+inline bool secure_getenv(const char* key, std::string* text) {
```

Comment:
We may want to rename this if it no longer matches the `secure_getenv(3)` signature